### PR TITLE
Disable test for CM that fails until the snapshot is ready

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: x-pack/libbeat/tests/system/test_management.py [hunk 1]
```diff
@@ -255,6 +257,7 @@ def create_and_assing_tag(self, blocks):
         url = self.get_kibana_url() + "/api/beats/agents_tags/assignments"
         r = requests.post(url, json=data, headers=headers,
                           auth=(self.es_user, self.es_pass))
+        print(r.text)
```
Comment (ycombinator) [inline x-pack/libbeat/tests/system/test_management.py]: Do you need this or just leftover from debugging?

File: x-pack/libbeat/tests/system/test_management.py [hunk 2]
```diff
@@ -152,8 +153,9 @@ def test_fetch_configs(self):
 
         proc.check_kill_and_wait()
 
-    @unittest.skipIf(not INTEGRATION_TESTS,
-                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
+    # @unittest.skipIf(not INTEGRATION_TESTS,
+    #                  "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
```
Comment (ycombinator) [inline x-pack/libbeat/tests/system/test_management.py]: Do these 2 lines need to be commented out?

File: x-pack/libbeat/tests/system/test_management.py [hunk 3]
```diff
@@ -85,8 +85,9 @@ def test_enroll_bad_pw(self):
         new_content = open(config_path, 'r').read()
         assert config_content == new_content
 
-    @unittest.skipIf(not INTEGRATION_TESTS,
-                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
+    # @unittest.skipIf(not INTEGRATION_TESTS,
+    #                  "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
```
Comment (ycombinator) [inline x-pack/libbeat/tests/system/test_management.py]: Do these 2 lines need to be commented out?

Comment (ph) [reply x-pack/libbeat/tests/system/test_management.py]: they will be back when the skip is removed

File: x-pack/libbeat/tests/system/test_management.py (hunk 2 above)
Comment (ph) [reply x-pack/libbeat/tests/system/test_management.py]: they will be back when the skip is removed

**Pull request merged**